hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/cache/topic/NotificationBasedTopicsCache.java

```diff
@@ -92,6 +94,11 @@ public void onTopicUnblacklisted(String qualifiedTopicName) {
     }
 
     @Override
+    public List<CachedTopic> getTopics() {
```

Comment:
OK we talked offline, so due to the fact our `TopicCache` implementation is Zookeeper-notification driven, and the underlying map may change anytime, we're leave above implementation that simply returns current snapshot of `CachedTopics`. Otherwise `ConcurrentModificationException` could be thrown when using an iterator.